ibmcloud/README-ZVSI.md

```diff
@@ -0,0 +1,350 @@
+# Setup procedure for IBM TEST Cloud (Staging environment)
```

Comment:
Given that the IBM Test Cloud information isn't widely relevant to open source and zVSI are creating on production I wonder if it is better to instead contribute some of the general fixes and improvements into README.md and add a little section on the differences e.g. in the Create a Kubernetes cluster section, after the instructions to add variables to the `terraform.tfvars` files you could add something like:
> **Note:** in order to create the cluster based on a different type of VSI image you can overwrite more parameters here e.g. to create an `s390x` based cluster add
>  ```
>  instance_profile_name = "bz2-2x8"
>  image_name = "ibm-ubuntu-18-04-1-minimal-s390x-3"
>  ```
That would stop us having to maintain two file with very similar content and still provide some zVSI instructions that should work?

Reply:
Agreed.
Merged some zVSI instructions to README.md so that we only need maintain one document

Reply:
Great. Thanks!